devlink_test.go

```diff
@@ -75,10 +77,20 @@ func TestDevLinkAddDelSfPort(t *testing.T) {
 		t.Fatal(err2)
 		return
 	}
-	t.Log(*port)
+	t.Logf("Port: %+v", *port)
 	if port.Fn != nil {
-		t.Log("function attributes = ", *port.Fn)
+		t.Logf("Port Function Attributes: %+v", *port.Fn)
+	}
+
+	if port.PortCap != nil {
+		t.Logf("Port Function Cap: %+v", *port.PortCap)
 	}
+
+	assert := assert.New(t)
+	assert.Equal(port.Controller, uint32(0))
```

Comment:
in the assert you should provide expected first and actual  see [1] 
Also it will be good to add message as third arg so it will be easy to debug. 

[1] - https://github.com/stretchr/testify/blob/v1.8.2/assert/assertions.go#L334

Reply:
done